src/Microsoft.IdentityModel.Validators/AadTokenValidationParametersExtension.cs

```diff
@@ -17,21 +17,51 @@ namespace Microsoft.IdentityModel.Validators
     /// </summary>
     public static class AadTokenValidationParametersExtension
     {
+        /// <summary>
+        /// Enables the validation of the signing keys used by the Microsoft identity platform (AAD) against the token.
```

Comment:
Perhaps something like: Enables validation of Microsoft Entra ID token signing keys.

Reply:
fixed